projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -474,109 +483,110 @@ function wpcom_display_global_styles_launch_bar( $bar_controls ) {
 		$preview_location = remove_query_arg( 'hide-global-styles' );
 	}
 
-	ob_start(); ?>
```

Comment:
The change here is better seen with "white space changes disabled" (as it's mostly indentation changes and addition of a wrapper)